scripts/partition_manager.py

```diff
@@ -922,8 +922,8 @@ def load_static_configuration(args, pm_config):
     # This is done since all partitions in pm_config will be resolved.
     for statically_defined_image in static_config:
         if statically_defined_image in pm_config and statically_defined_image:
-            print(f"Dropping partition '{statically_defined_image}' "
-                  f"since it is statically defined.")
+            print(f"Partition '{statically_defined_image}' is not included "
+                  f"in the dynamic resolving since it is statically defined.")
```

Comment:
Why an `f` here ?
```suggestion
                  "in the dynamic resolving since it is statically defined.")
```

Reply:
I think it looks more consistent that way.

Reply:
when I read such code, it give me the impression of code copied from elsewhere, and user not knowing what `f` means in this context. It just happened to be there :stuck_out_tongue: